drafts/draft-lcurley-moq-use-cases.md

```
@@ -0,0 +1,438 @@
+~~~
+
+## Upstream
+All of these separate viewers could be watching the same broadcast.
+How is a relay supposed to fetch the content from upstream?
+
+moq-lite addresses this by providing the publisher's Track Priority and Group Order in the INFO message.
+This is the intended behavior for the first hop and dictates which viewers are preferred.
+
+For example, suppose the producer chooses:
+
+~~~
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)